seokhwan/2/가운데_글자_가져오기.js

```diff
@@ -0,0 +1,8 @@
+function solution(s) {
+  let answer = "";
+  let i = parseInt(s.length / 2);
+
+  if (s.length % 2) answer = s[i];
+  else answer = s[i - 1] + s[i];
```

Comment:
```suggestion
  if (s.length % 2) {
      answer = s[i];
  } else {
      answer = s[i - 1] + s[i];
  };```

이런 식으로 중괄호로 묶어 주는게 가독성에 좋다는 것을 알려 드리고 싶네요~~